templates/user/profile.tmpl

```diff
@@ -17,6 +17,9 @@
 						{{if .Owner.FullName}}<span class="header text center">{{.Owner.FullName}}</span>{{end}}
 						<span class="username text center">{{.Owner.Name}}</span>
 						<a href="{{.Owner.HomeLink}}.rss"><i class="ui grey icon tooltip ml-3" data-content="{{.i18n.Tr "rss_feed"}}" data-position="bottom center">{{svg "octicon-rss" 18}}</i></a>
+						<i class="ui grey icon tooltip ml-3">{{svg "octicon-person" 18}}</i>
+						<a href="{{.Owner.HomeLink}}?tab=followers">{{.Owner.NumFollowers}} {{.i18n.Tr "user.followers"}}</a> .
+						<a href="{{.Owner.HomeLink}}?tab=following">{{.Owner.NumFollowing}} {{.i18n.Tr "user.following"}}</a>
```

Comment:
```suggestion
						<a class="muted" href="{{.Owner.HomeLink}}?tab=followers">{{svg "octicon-person" 18 "mr-2"}}{{.Owner.NumFollowers}} {{.i18n.Tr "user.followers"}}</a> · <a class="muted" href="{{.Owner.HomeLink}}?tab=following">{{.Owner.NumFollowing}} {{.i18n.Tr "user.following"}}</a>
```

<img width="256" alt="image" src="https://user-images.githubusercontent.com/115237/169394379-4b5ede66-f70d-4424-9d07-d3edb9d4edd2.png">

Reply:
Thank you for your suggestion. I really needed the "muted" class.

Reply:
Is there anything missing? The UI on try.gitea.io is not the same as the screenshot.